pkg/agent/openflow/pipeline.go

```diff
@@ -1669,7 +1729,12 @@ func (f *featurePodConnectivity) ipv6Flows() []binding.Flow {
 
 // conjunctionActionFlow generates the flow to jump to a specific table if policyRuleConjunction ID is matched. Priority of
 // conjunctionActionFlow is created at priorityLow for k8s network policies, and *priority assigned by PriorityAssigner for AntreaPolicy.
-func (f *featureNetworkPolicy) conjunctionActionFlow(conjunctionID uint32, table binding.Table, nextTable uint8, priority *uint16, enableLogging bool) []binding.Flow {
+func (f *featureNetworkPolicy) conjunctionActionFlow(conjunctionID uint32, table binding.Table, nextTable uint8,
+	priority *uint16, enableLogging, isIngress, isIGMP bool) []binding.Flow {
+	tableID := table.GetID()
+	if f.isMulticastIngressRule(tableID, isIngress, isIGMP) || f.isIGMPEgressRule(tableID, isIngress, isIGMP) {
```

Comment:
Please add comments to explain the behavior.

Reply:
done